### DISCUSSION TITLE
I have 2 folders of localisation in my pc?

### DISCUSSION THREAD

**Discussion by Gonzalo-Bruna** [ppy/osu#14477] [Q&A] [answered]
After last update I seem to have 2 folders for localisation for each version of lazer, not sure if this is an intented behaviour or maybe is just something on my end. (I'm guessing they're localisation folders due to what's inside is a bunch of folders which seems to be languages abbreviations)

![image](https://user-images.githubusercontent.com/41755816/130631468-57a78cb6-bdbf-4f55-94dc-ae343013de9f.png)

**Comment by frenzibyte** [comment]
They're not localisation folders, they're the game's own files on different versions. You mention what's inside of them are "localisation folders" only, is that correct? or are there further files in the same folder?

**Reply by Gonzalo-Bruna** [reply]
I can confirm there are more files than just those folders I told you, but then, why do I have files of 2 different versions? since each version's weight is nearly 300mb, maybe is a waste of space (or not), that's why I'm asking. Why do I have a folder of an older version ?

**Comment by Game4all** [comment, marked as answer]
This has to do with how Squirrel works and can't be fixed (for the moment). See #12601

**Reply by peppy** [reply]
it doesn't even need to be fixed. it's intentional and allows rolling back to a previous version
